docs/configuration-ini.md

```diff
@@ -255,6 +255,39 @@ bias_initializer = zeros
 Activation = softmax
 ```
 
+### Backbone section
+
+Describe another model, termed as backbone, to be used used in the model described by the current ini file. The backbone to be used can described with another ini configuration file path, or with model file for external frameworks. Support for backbones of external framework for Tensorflow-Lite is provided natively with Tensorflow-Lite framework. Support for backbones of other external frameworks is done using nnstreamer and its plugin. When using nnstreamer for external framework, ensure to add the corresponding baseline ML framework and its corresponding nnstreamer plugin as a dependency or install manually. For example, when using PyTorch based model as a backbone, both the packages *PyTorch* and *nnstreamer-pytorch*.
```

Comment:
Could you have one sentence in a line? Github doesn't allow me to review part by part :(

Reply:
I will update.

Reply:
done.